scripts/_helpers.py

```diff
@@ -785,3 +785,7 @@ def get_snapshots(snapshots, drop_leap_day=False, freq="h", **kwargs):
         time = time[~((time.month == 2) & (time.day == 29))]
 
     return time
+
+
+def get_country_from_node_name(node_name: str) -> str:
+    return node_name[:2]
```

Comment:
you can use the function [`assign_locations`](https://github.com/PyPSA/pypsa-eur/blob/f8d0efbe992413aac522851939dd7445f6084595/scripts/make_summary.py#L29)

Reply:
Yes, `assign_locations` seems to do what I need. Though it looks like it requires a `pypsa.Network` as input. Which network would I pass?